roles/openshift_master_upgrade/handlers/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Maybe too wild but: https://paste.fedoraproject.org/paste/yYpTUUuwZl~sB7bORsieUl5M1UNdIGYhyRLivL9gydE=

The point:
1) Every handler is actually a task that could be needed at any point of any role. So make it includable via ``include_role: name=ROLE file=FILE``
2) in handlers, actually include handler's body as a task from the same role (thus the symlink)
3) roles that need ROLEH's handlers, create new handlers that run tasks from ROLEH's handlers

Benefits:
* no more handlers duplication
* you know precisely from where a handler of a different role comes from
* role does not include files out of the role -> the role can be still shared on ansible-galaxy

Drawbacks:
* symlinks from handlers to tasks
* duplication of handlers on a level of defining handlers and include[_role] statements
* you can not specify ROLEH as a dependency of your role